src/components/User/UserMain/Notice/NoticeDetailModal/index.tsx

```
@@ -0,0 +1,70 @@
+            <S.CloseIcon>✕</S.CloseIcon>
+          </S.CloseButton>
+        </S.ModalHeader>
+
+        <S.ModalContent>
+          <S.NoticeMetadata>
+            <S.NoticeDate>작성일: {formatDate(notice.createdAt)}</S.NoticeDate>
+          </S.NoticeMetadata>
+
+          <S.NoticeContentText
+            dangerouslySetInnerHTML={{
+              __html: notice.content.replace(/\n/g, '<br/>')
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)